app/components/c-product-detail.vue

```diff
@@ -40,9 +40,22 @@
         </div>
         <div :class="b('details')">
           <c-collapse-group>
-            <c-collapse v-if="product.tech_attributes" :title="$t('c-product-detail.technicalDataTitle')">
+            <c-collapse v-if="product.tech_attributes"
+                        :title="$t('c-product-detail.technicalDataTitle')"
+            >
               <c-attribute-grid :attributes="product.tech_attributes"/>
             </c-collapse>
+            <c-collapse v-if="hasMedia"
+                        :class="b('documents')"
+                        :background="false"
```

Comment:
- [ ] I think the content should have here also a background, that it looks like the technical data section - but please check this with designer (Christoph Hirschbühl)